.github/workflows/build-and-test-pr.yml

```
@@ -147,19 +179,19 @@ jobs:
     strategy:
       fail-fast: false
     env:
-      ZOOKEEPER_VERSION: 3.6.3
+      ZOOKEEPER_VERSION: 3.7.2
     steps:
-      - uses: actions/cache@v3
-        name: "Cache zookeeper binary archive"
+      - name: "Cache zookeeper binary archive"
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)